notebooks/ArynPartitionerPython.ipynb

```diff
@@ -71,7 +71,9 @@
    "id": "4",
    "metadata": {},
    "source": [
-    "Replace the `YOUR-ARYN-API-KEY` with your key "
+    "Replace the `YOUR-ARYN-API-KEY` with your key and execute the export command\n",
```

Comment:
This is still flawed. It requires people to put their key in the notebook content itself. 
Think of the key as a password. You wouldn't put your password in here and commit it to github, so you shouldn't encourage people to put the password in the same place.

The safe way to get your key into the environment is something like:
```
% ARYN_API_KEY=... jupyter lab
```

Reply:
yes, this was not clear. I will remove this entirely and add your suggestion which is much more clear and concise.

```from sycamore.utils.aryn_config import ArynConfig, _DEFAULT_PATH
assert ArynConfig.get_aryn_api_key() != "", f"Unable to find aryn API key.  Looked in {_DEFAULT_PATH}"
# if the above assertion fails, you can either set the environment variable ARYN_API_KEY and restart jupyter
# or make a yaml file at the specified path in the assertion error that looks like the next line with the leading '#':
# aryn_token: "YOUR-ARYN-API-KEY"

# It is unsafe, but if neither of those options work, you can put it in this notebook with
# import os
# os.environ["ARYN_API_KEY"] = "UNSAFE-ARYN-API-KEY-LOCATION"
# but beware that it is easy to accidentally commit the notebook file and have it include your key.```